snuba/datasets/outcomes_raw.py

```diff
@@ -2,57 +2,19 @@
 from typing import Mapping, Sequence
 
 from snuba.datasets.dataset import TimeSeriesDataset
-from snuba.clickhouse.columns import (
-    ColumnSet,
-    DateTime,
-    LowCardinality,
-    Nullable,
-    String,
-    UInt,
-    UUID,
-)
-from snuba.datasets.schemas.tables import MergeTreeSchema
-from snuba.datasets.dataset_schemas import StorageSchemas
+from snuba.datasets.storages.factory import get_storage
 from snuba.datasets.plans.single_storage import SingleStorageQueryPlanBuilder
-from snuba.datasets.storage import WritableTableStorage
 from snuba.query.extensions import QueryExtension
 from snuba.query.organization_extension import OrganizationExtension
 from snuba.query.processors.basic_functions import BasicFunctionsProcessor
-from snuba.query.processors.prewhere import PrewhereProcessor
 from snuba.query.query_processor import QueryProcessor
 from snuba.query.timeseries import TimeSeriesExtension
 
 
 class OutcomesRawDataset(TimeSeriesDataset):
     def __init__(self) -> None:
-        read_columns = ColumnSet(
-            [
-                ("org_id", UInt(64)),
-                ("project_id", UInt(64)),
-                ("key_id", Nullable(UInt(64))),
-                ("timestamp", DateTime()),
-                ("outcome", UInt(8)),
-                ("reason", LowCardinality(Nullable(String()))),
-                ("event_id", Nullable(UUID())),
-            ]
-        )
-
-        read_schema = MergeTreeSchema(
-            columns=read_columns,
-            local_table_name="outcomes_raw_local",
-            dist_table_name="outcomes_raw_dist",
-            order_by="(org_id, project_id, timestamp)",
-            partition_by="(toMonday(timestamp))",
-            settings={"index_granularity": 16384},
-        )
-
-        storage = WritableTableStorage(
-            schemas=StorageSchemas(
-                read_schema=read_schema, write_schema=None, intermediary_schemas=[]
```

Comment:
I'm not actually sure why we have a WritableTableStorage without a write schema or table writer here. This has changed since it's using the same table storage as outcomes now.

Reply:
It was a mistake.